system_tests/utils.go

```diff
@@ -428,6 +433,28 @@ func newRabbitmqCluster(namespace, instanceName string) *rabbitmqv1beta1.Rabbitm
 	return cluster
 }
 
+func overrideSecurityContextForOpenshift(cluster *rabbitmqv1beta1.RabbitmqCluster) *rabbitmqv1beta1.RabbitmqCluster {
```

Comment:
I think this doesn't need to return the pointer that's passed in - you can just modify the RabbitmqCluster object referenced via the pointer and return nothing.